enet.pyx

```diff
@@ -252,7 +252,7 @@ cdef class Address:
     def __init__(self, host, port):
         if host is not None:
             # Convert the hostname to a byte string if needed
-            self.host = host if isinstance(host, bytes) else bytes(host, "ascii")
+            self.host = host if isinstance(host, bytes) else host.encode('cp437')
```

Comment:
cp437 is extended ascii, simply an encoding which covers all the possible bytes. I think this is what you want in case of python3? Because in py2k it is a "string" of bytes, i.e. str type. Aka py2k byte type. 

At one point or another self.host needs to be a str (in both py3k and py2k) so that's a workaround around that in py3k.

IIRC